core/utils.py

```diff
@@ -272,6 +273,22 @@ def normalizeKey(key: Union[None, 'db.KeyClass']) -> Union[None, 'db.KeyClass']:
     return db.Key(key.kind, key.id_or_name, parent=parent)
 
 
+def strings_equal(s1: str, s2: str) -> bool:
```

Comment:
@AchimSchumacher thanks for solving this issue!

You've added the function `strings_equal` here. IMHO, does just using hmac.compare_digest not satisfy already? What's the purpose of the normalization? I want to keep the amount of self-implemented security-related utility functions low, if there's already a valid implementation in the standard library.

Reply:
Yes, you are right, hmac.compare_digest would serve the same purpose. Please remove strings_equal.